### ISSUE TITLE
tilers attribute error

### ISSUE THREAD

**Issue by gioleppe** [scikit-mobility/scikit-mobility#93]
Hi, 

I stumbled upon an attribute error while running your tutorial code for tdf to fdf conversion.

Running the line
`tessellation = tilers.tiler.get("squared", base_shape="Beijing, China", meters=15000)`

resulted in this error traceback:
```
AttributeError                            Traceback (most recent call last)
<ipython-input-38-d33a2564af0f> in <module>
      1 from skmob.tessellation import tilers
      2 
----> 3 tessellation = tilers.tiler.get("squared", base_shape="Beijing, China", meters=500)
      4 
      5 print(tdf.head())

~/opt/anaconda3/envs/skmob/lib/python3.7/site-packages/scikit_mobility-1.0-py3.7.egg/skmob/tessellation/tilers.py in get(self, service_id, **kwargs)
     29     def get(self, service_id, **kwargs):
     30 
---> 31         return self.create(service_id, **kwargs)
     32 
     33 

~/opt/anaconda3/envs/skmob/lib/python3.7/site-packages/scikit_mobility-1.0-py3.7.egg/skmob/tessellation/tilers.py in create(self, key, **kwargs)
     25             raise ValueError(key)
     26 
---> 27         return tiler(**kwargs)
     28 
     29     def get(self, service_id, **kwargs):

~/opt/anaconda3/envs/skmob/lib/python3.7/site-packages/scikit_mobility-1.0-py3.7.egg/skmob/tessellation/tilers.py in __call__(self, base_shape, meters, crs, window_size)
    110                 raise ValueError("Not valid base_shape object. Accepted types are str, GeoDataFrame or GeoSeries.")
    111 
--> 112         return self._build(base_shape, meters, crs)
    113 
    114     def _build(self, base_shape, meters, crs=constants.DEFAULT_CRS):

~/opt/anaconda3/envs/skmob/lib/python3.7/site-packages/scikit_mobility-1.0-py3.7.egg/skmob/tessellation/tilers.py in _build(self, base_shape, meters, crs)
    169         gdf[constants.TILE_ID] = gdf[constants.TILE_ID].astype('str')
    170 
--> 171         return gdf.to_crs(crs)
    172 
    173 

~/opt/anaconda3/envs/skmob/lib/python3.7/site-packages/geopandas/geodataframe.py in to_crs(self, crs, epsg, inplace)
    562         else:
    563             df = self.copy()
--> 564         geom = df.geometry.to_crs(crs=crs, epsg=epsg)
    565         df.geometry = geom
    566         df.crs = geom.crs

~/opt/anaconda3/envs/skmob/lib/python3.7/site-packages/pandas/core/generic.py in __getattr__(self, name)
   5272             if self._info_axis._can_hold_identifiers_and_holds_name(name):
   5273                 return self[name]
-> 5274             return object.__getattribute__(self, name)
   5275 
   5276     def __setattr__(self, name: str, value) -> None:

~/opt/anaconda3/envs/skmob/lib/python3.7/site-packages/geopandas/geodataframe.py in _get_geometry(self)
    101             raise AttributeError(
    102                 "No geometry data set yet (expected in"
--> 103                 " column '%s'." % self._geometry_column_name
    104             )
    105         return self[self._geometry_column_name]

AttributeError: No geometry data set yet (expected in column 'geometry'.
```

It seems like the geometry column is empty.
Running the same line with a different query string such as "Bologna, Italy" works just fine.
I guess the problem could be in tilers.py or, more specifically, in the bbox_from_name function call to utils.py.
This is just a guess since I saw you changed the implementation to that part by removing the dependency to osnmx a while back.

Thanks for your work on this great library.

**Comment by gioleppe** [comment]
By further analysing the issue it seems like it is apparent only when choosing a place in China.
Every other locations works fine

**Comment by gbarlacchi** [comment]
Thanks for reporting this. Unofrtunately this is a problem with the inverse geocoding from OSM. that we dind't solve yet. For some cities, they return empty geometries and thus we are not able to build the grid.

**Comment by gioleppe** [comment]
Have you tried reading this?

https://github.com/gboeing/osmnx/issues/16
Choosing the second result with parameter "which_result" fixed the issue on osmnx.
To my knowledge is impossible to add the parameter to the library's tiler.get function call.

in the definition line for "bbox_from_name" you set which_result to 0, and the value is never passed from the tiler function to bbox_from_name.

`def bbox_from_name(query, which_result=0, crs=None):`

this is the workaround I've found: 

`from skmob.tessellation import tilers
import osmnx as ox

G = ox.graph_from_place('Beijing, China', which_result=2, network_type='drive')
ox.plot_graph(G)

nodes, _ = ox.graph_to_gdfs(G)
nodes.head()

tessellation = tilers.tiler.get("squared", base_shape=nodes, meters=500)
print(tessellation)
`
First of all I get the graph from osmnx graph_from_place, 
then I convert the graph to a geodataframe,
finally I pass the geodataframe to scikit mob's tiler.

Hope I've been useful, thanks again for your work on the library

**Comment by FilippoSimini** [comment]
This will be fixed in release 1.1.2, where it will return the first Polygon or Multipolygon object returned by OSM.
